package.json

```diff
@@ -60,6 +60,9 @@
     "vite-plugin-dts": "4.5.3",
     "vitest": "^3.0.8"
   },
+  "overrides": {
+    "react-json-view-lite": "2.4.1"
+  },
```

Comment:
This was causing a warning when doing an install because one of the docs packages users an old version, this don't affect any of the packages